executor/poolmgr/gp.go

```diff
@@ -363,6 +363,13 @@ func (gp *GenericPool) specializePod(pod *apiv1.Pod, metadata *metav1.ObjectMeta
 		return err
 	}
 
+	env, err := gp.fissionClient.
```

Comment:
Don't add this to specialize; it slows down cold starts.  Move it to the gp struct construction